synapse/storage/databases/state/bg_updates.py

```diff
@@ -98,153 +98,100 @@ def _get_state_groups_from_groups_txn(
             # a temporary hack until we can add the right indices in
             txn.execute("SET LOCAL enable_seqscan=off")
```

Comment:
This setting is Postgres specific so it's the only thing left that is gated.

Do we have to worry about this in SQLite? I don't see a similar setting for SQLite